Add script for running large batches data

Lostpart opened the pull request:
Add script for running large batches data
Some minor change on structure consistency
Some other minor change

Lostpart committed:
Add script for running large batches data
Add a daily script for running new data generate everyday

Lostpart committed:
Add comment

Lostpart committed:
Change the way how way_types and way_type_avg_speed_limit generate

Lostpart committed:
Change some print to only print in debug mode

Lostpart committed:
Update places where use path to use a global variable in global_var.py
Change some function and file that use to run every folder in data/ folder to only run at given day

Lostpart committed:
Update requirements.txt

Lostpart committed:
Update README.md

Lostpart committed:
Update the way to predict road speed for road that has no data

Lostpart committed:
Add auto clean old logs feature in daily_script.sh

Lostpart committed:
Fixed some minor error

wdzhong commented on script/generate_prediction_in_large_batches.py:
```diff
@@ -0,0 +1,135 @@
+
+def predict_speed_dict_to_json(predict_speed_dict, target_date_str, time_slot_interval, interval_idx,
+                               final_way_table, way_types, way_type_avg_speed_limit,
+                               save_path="cache/predict_result/{0}/{1}/{2}.json"):
```
Why do you use this format in parameters? Can we just pass in the formatted (final) string? This happens in a few of other places as well.

Lostpart committed:
Add comment

Lostpart committed:
Fix a minor issue
Now the program will handle additional unknown columns, instead of stop running.

Lostpart committed:
Fix a minor issue
Program may stop running when input file only have one line.

Lostpart committed:
Delete the sorted folder before put and new file into the sorted folder

Lostpart committed:
Update files of data/yyyyMMdd/result/ folder, now include 15 min interval files and .p pickle file for both 5 and 15 min interval

Lostpart committed:
Include way_id in cache/predict_result
